pytest_mpl/plugin.py

```diff
@@ -109,10 +147,15 @@ def pytest_addoption(parser):
     group.addoption('--mpl-hash-library',
                     help="json library of image hashes, relative to "
                     "location where py.test is run", action='store')
+    group.addoption('--mpl-generate-summary', action='store_true',
```

Comment:
I wonder if we want to allow for non-HTML summaries in future. Can we easily extend this to allow e.g. ``--mpl-generate-summary=xml`` and have the one without a format default to HTML? (if so then we can extend this in a back-compat way in future)

Reply:
This is actually a good call, I was thinking of a json summary with a fancy (static) html / js frontend

Reply:
I think the description needs updating to mention that the format should be given? Also maybe validate the formats?